packages/accordion/src/AccordionTitle.tsx

```diff
@@ -0,0 +1,56 @@
+import { animated, useSpring } from 'react-spring';
+import { Box, Flex } from '@ag.ds-next/box';
+import { Icon } from '@ag.ds-next/icon';
+import { usePrefersReducedMotion } from '@ag.ds-next/core';
+
+export type AccordionTitleProps = {
+	id: string;
+	ariaControls: string;
+	isOpen?: boolean;
+	onToggle: () => void;
+	titleHeadingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Comment:
```suggestion
	level?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Reply:
We can simplify the name here 👍 You can keep the prop name `titleHeadingLevel` on the `Accordion` component

Reply:
Maybe `tag` and `titleHeadingTag` would be a better name 🤔